docs/howto/kubernetes/gardener/create-shoot-cluster.md

```diff
@@ -0,0 +1,113 @@
+# Create Kubernetes cluster using Cleura Cloud Control Panel 
+<!-- Config interpolation for the gui name didn't work for the title here. -->
+
+If you want to create a Kubernetes cluster, you can do it via {{config.extra.gui}} using Gardener.
+This tutorial shows you how to do that, and how to deploy a sample application on such cluster.
+
+## Prerequisites
+
+To access the cluster from your computer, you will need [kubectl](https://kubernetes.io/docs/tasks/tools/) installed on your machine.
+
+## Creating Kubernetes cluster in {{config.extra.gui}}
+
+To get started, navigate to [{{config.extra.gui_domain}}](https://{{config.extra.gui_domain}}) and in the side panel choose Kubernetes
+-> [Managed Kubernetes](https://{{config.extra.gui_domain}}/containers/gardener).
```

Comment:
In the current configuration, the Markdown string `->` will *not* render as an arrow, `→`. To do that, we'd have to [enable the `smarty` Markdown extension](https://www.mkdocs.org/user-guide/configuration/#markdown_extensions), which we currently don't do. You probably just want to replace `->` with `→` here.